pages/references.xml

```diff
@@ -76,6 +81,141 @@ This manuscript is in the same hand as <ref type="hand" corresp="ESap002#h2"/>
          <p>Please specify the <att>type</att> for each
     reference. Allowed values are encoded in the schema.</p>
 </div>
+         
+         <div type="level2">
+            <head>References to text passages</head>
+            <p>You may use references to parts of a text, in some cases, like <gi>locus</gi>, 
```

Comment:
I think this is again confusing because of my use of "resolve". I have added a piece of sentence to clarify that you do not need a div, a pb, a lb, or anything for locus, where you are adding a reference to a real word structure. a different issue, and the one I want to outline here also with the example, is the reuse of structures of reference which do not have a material correspondent in real life and only exist in the encoding and thus need to be there and looked after.